hub/apps/get-started/dev-options.md

```diff
@@ -17,20 +17,21 @@ There is a wide range of options for developing applications for Windows. The be
 
 To read more about each of these Windows app development options, see [Writing apps for Windows](index.md).
 
-| Feature | .NET MAUI | React Native (RNW) | UWP | Win32 | Windows Forms | WinUI 3 | WPF |
+| Feature | .NET MAUI | React Native (RNW) | UWP XAML (Windows.UI.Xaml) | Win32 | Windows Forms | WinUI 3 | WPF |
```

Comment:
This is tricky, but the team is willing to accept this change and see how it's received. There isn't an easy way to compare these options and pick out just the "UI framework" part of each one. .NET MAUI is much more than a UI framework.

Reply:
That's true, but the problem is that there's rows for UI-Framework-specific things (e.g the designer and Fluent Design) and these don't apply to non-XAML UWP apps like DX, Composition, or Flutter ones

Reply:
> This is tricky, but the team is willing to accept this change and see how it's received. There isn't an easy way to compare these options and pick out just the "UI framework" part of each one. .NET MAUI is much more than a UI framework.

It should not be tricky. 'UWP' should only refer to the sandboxed application model (CoreApplication) and it can run multiple UI frameworks on top. Unfortunately, Microsoft has had a long history of using confusing and conflated terminology.